main/src/ca/uwaterloo/flix/language/phase/unification/BoolFormulaTable.scala

```diff
@@ -66,19 +71,19 @@ object BoolAlgebraTable {
     *
     * The table is pre-computed and initialized when this class is loaded.
     */
-  private lazy val Table: Array[BoolAlgebra] = initTable()
+  private lazy val Table: Array[ExplicitFormula] = initTable()
 
   /**
     * Attempts to minimize the given formula `f`.
     *
     * Returns the same formula or a smaller formula.
     */
-  def minimizeFormula(f: BoolAlgebra): BoolAlgebra = {
+  def minimizeFormula(f: ExplicitFormula): ExplicitFormula = {
     // Compute the number of free variables.
     val numVars = f.freeVars.size
 
     // Determine whether to minimize once or recursively.
-    val result = if (numVars <= MaxVars) {
+    val result = if (f.freeVars.subsetOf(VarSet)) {
```

Comment:
I still don't get this :)

Reply:
The formula table expects that the variables in the formula have the numbers 0, 1, 2, 3
Previously, it was true by construction that if you had 4 variables in your formula, then they must be 0, 1, 2, 3
This is no longer true, due to the way Type -> BoolFormula works. Therefore, we need to perform alpha conversion unless we happen to be in the nice world of 0, 1, 2, 3.

That said, I need to revisit this since needing to alpha convert != needing to minimize recursively

Reply:
But why does Type -> BoolFormula not name them 0 to n?

Reply:
It does. But say we have 10 variables total. We do transformations in the SVE and then we have to minimize a type with vars 1, 3, 5, 9. So we have to do alpha-renaming.

In the old implementation, we do Type -> BoolFormula at this point that includes the mapping 1, 3, 5, 9 -> 0, 1, 2, 3

In the new implementation, we already have a Boolean formula at this point, so we need an extra alpha-renaming step.

Reply:
I see OK.

Reply:
Could you add some comments.